### PR TITLE
feat: extract env vars

### PR DESCRIPTION
**Pull request by morrieinmaas**
Refactor out constants and env vars.
    
This also includes enabling changing the routes/enpoints via the env file instead of manually having to do so in the code.

### CODE REVIEW

**Comment by morrieinmaas** [comment]
closes #167

**Review by TimoGlastra — COMMENTED** [review]
Great for picking up this issue. I'm not sure however if we should extract the endpoint and tags into environment variables.  Do you have a specific use case that would require changing the tags/endpoints dynamically? I think it makes it harder to understand the code as you need to follow the environment variables to know for sure what the endpoint should be

Interested to hear your opinion on this

**Comment by morrieinmaas** [comment]
hmm fair point. I'm happy to not do it that way. Reason why I did that is that we have changed the namings and routes 2 already in the past. I thought this'd make it easier (also for non-YOMA) to make the routing and tagging the way they like. But again, fine by me to not do it that way.

File: environments/yoma-ga/fastapi.default.env [hunk 1]
```diff
@@ -15,4 +15,50 @@ ACAPY_TAILS_SERVER_BASE_URL=http://tails-server:6543
 LEDGER_NETWORK_URL=http://ledger-browser:8000/register
 LEDGER_TYPE=von
 
-PYTHONPATH=/
\ No newline at end of file
+# TEST RELATED
+TEST_LEDGER_URL=http://localhost:9000/register"
+CONFTEST_ADMIN_URL = os.getenv("CONFTEST_ADMIN_URL", "http://localhost")
```
Comment (TimoGlastra) [inline environments/yoma-ga/fastapi.default.env:20]:
```suggestion
CONFTEST_ADMIN_URL = "http://localhost"
```

**Comment by morrieinmaas** [comment]
Sure. sounds good

**Review by TimoGlastra — APPROVED** [review]

**Pull request merged**